src/forms/user_form.py

```diff
@@ -84,9 +84,12 @@ class UserInfoForm(FlaskForm):
                 field_class = StringField
             if field.get('type') == 'textarea':
                 field_class = TextAreaField
-            elif field.get('type') == 'integer':
+            if field.get('type') == 'integer':
```

Comment:
Is this change correct?

Reply:
I think yes, there is an other new condition below this one

Reply:
We could use `match` and `case` for all these conditions ?

Reply:
Done it in 00238f12be35f88a7348b561a92bfb9d583d756b